mover-syncthing/Dockerfile

```
@@ -0,0 +1,85 @@
+# Build the Syncthing image from Golang
+FROM golang:1.17 AS syncthing-builder
+
+# source directory
+RUN mkdir -p /workspace
+WORKDIR /workspace
+
+# latest release as of writing this
+ARG SYNCTHING_VERSION="v1.18.4"
+ARG SYNCTHING_GIT_HASH="951b0589522e13e14e8b40d33ea66de1e69c8b4e"
+
+# clone & cd
+RUN git clone \
+  --depth 1 \
+  --branch=${SYNCTHING_VERSION} \
+  https://github.com/syncthing/syncthing.git
+WORKDIR /workspace/syncthing
+
+# Make sure we have the correct Syncthing release
+RUN /bin/bash -c "[[ $(git rev-list -n 1 HEAD) == ${SYNCTHING_GIT_HASH} ]]"
+# Create the binary for syncthing 
+RUN go run build.go
+
+
+ENTRYPOINT [ "/workspace/syncthing/bin/syncthing" ]
+
+# Build final container 
+FROM registry.access.redhat.com/ubi8-minimal
+
+# RUN useradd -ms /bin/bash stuser
+# USER root
+
+RUN microdnf update -y && \
+    microdnf clean all && \
+    rm -rf /var/cache/yum
+
+EXPOSE 21027/udp 2000/tcp 22000 8384/tcp
+
+COPY --from=syncthing-builder \
```

To match the L2 change
```suggestion
COPY --from=builder \
```